jupyterlab_bigquery/src/components/details_panel/service/list_model_details.ts

```diff
@@ -24,6 +25,35 @@ export interface ModelDetails {
   details: ModelDetailsObject;
 }
 
+interface TrainingRunDetailsObject {
+  actual_iterations?: number; // int
+  data_split_column?: any;
+  data_split_eval_fraction?: number; // float
+  data_split_method?: string; // model enum
+  distance_type?: string; // model enum
+  early_stop?: string; // BoolValue to string
+  initial_learn_rate?: number; // float
+  input_label_columns?: string; // could be an array?
+  kmeans_initialization_column?: string;
+  kmeans_initialization_method?: string; // model enum
+  l1_regularization?: number; // DoubleValue
+  l2_regularization?: number; // DoubleValue
+  label_class_weights?: any; // _MODEL_TRAININGRUN_TRAININGOPTIONS_LABELCLASSWEIGHTSENTRY
+  learn_rate?: number; // float
+  learn_rate_strategy?: string; // model enum
+  loss_type?: string; // model enum
+  max_iterations?: number; // int
+  min_relative_progress?: number; // DoubleValue
+  model_uri?: string;
+  num_clusters?: number; // int
+  optimization_strategy?: string; // model enum
+  warm_start?: string; // BoolValue
+}
```

Comment:
out of all these options, I'm not sure what format `input_label_columns` and `label_class_weights` will come as, but I've labeled all of these particular types with more specific annotations just so its a little more clear where they come from.

Reply:
I searched internally. I think this can probably be either a string and array of strings